src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -130,7 +130,7 @@
                                       Style="{StaticResource InCanvasScrollViewerStyle}"
                                       CanContentScroll="True"
                                       Width="250">
-                            <ContentControl Template="{StaticResource MemberControlTemplate}" />
+                            <ContentControl Template="{StaticResource InCanvasMemberControlTemplate}" />
```

Comment:
It seems like right now we have both `MemberControlTemplate` and `InCanvasMemberControlTemplate` templates, is there a reason we don't apply this style to both the regular library search (in addition to the in-canvas search)? I'd be glad if we don't add another new template and keep our `XAML` simpler :)